src/dnp3_python/dnp3station/station_utils.py

```diff
@@ -19,33 +19,34 @@
 
 # alias
 ICollectionIndexedVal = Union[opendnp3.ICollectionIndexedAnalog,
-                              opendnp3.ICollectionIndexedBinary,
-                              opendnp3.ICollectionIndexedAnalogOutputStatus,
-                              opendnp3.ICollectionIndexedBinaryOutputStatus]
+opendnp3.ICollectionIndexedBinary,
```

Comment:
I am not sure I like the way this is formatted....did this go through the pre-commit formatting?

Reply:
It didn't use black as formatting. Reformatted accordingly.